src/import/Import.ml

```diff
@@ -21,6 +21,8 @@ include (
 include Option.Monad_infix
 include Stdio
 
+exception Formatting_disabled
+
```

Comment:
Could this exception be moved to `Fmt_ast` instead since it is really part of the interface of its `fmt_structure` (and `fmt_signature`) functions.